x-pack/plugins/canvas/canvas_plugin_src/strings/functions/function_help.ts

```diff
@@ -4,7 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { AvailableFunctions, Function } from '../../functions/types';
+import { ExpressionFunction } from 'src/legacy/core_plugins/interpreter/public';
+import { AvailableFunctions } from '../../functions/types';
```

Comment:
Nit: Is this an extraneous import? I don't see `AvailableFunctions` being used in this file.

Reply:
Line 108:

https://github.com/elastic/kibana/pull/37300/files/159bcd6d3875d971141c5d4dc6a1cf2cff3256b6#diff-a34b003bb151aba2727b3233b181b493R108

Reply:
Ah okay, forgot to expand. All good 👍